### ISSUE TITLE
Storage of PNG files (figures)

### ISSUE THREAD

**Issue by tobolar** [modelica/ModelicaStandardLibrary#2695]
There is a convention concerning figures' storage which states that
> The PNG files should be placed in a folder which exactly represents the package structure. 

(see also Modelica.UsersGuide.Conventions.Documentation.Format.Figures).

According to this, the figures collected in `Modelica/Resources/Images/UsersGuide` should be stored appropriately. This is already the case for figures used in Modelica.UsersGuide.Conventions, i.e. there is a directory `Modelica/Resources/Images/UsersGuide/Conventions` containing corresponding subdirectories.

**Comment by tobolar** [comment]
@AHaumer @christiankral I'm going to proceed here with Electrical and Magnetic in the next days. In order to omit conflicts please let me know when I could start. Or you can of course fix this issue for those two packages on your own.

**Comment by dietmarw** [comment]
As far as I can tell there are no more open PRs left that would conflict with changes to be done in Electrical and Magnetic

**Comment by tobolar** [comment]
You are right. But working on this issue is really time consuming and I will not find time to fix this till MSL 4.0.0 fix.

**Comment by dietmarw** [comment]
Yes. Hopefully @christiankral and @AHaumer can take care of those.

**Comment by christiankral** [comment]
OK then. @AHaumer I guess we should always indicate in advance what packages we are working on to avoid redundant work.

**Comment by beutlich** [comment]
> 
> 
> You are right. But working on this issue is really time consuming and I will not find time to fix this till MSL 4.0.0 fix.

Remvong milestone.
